CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+- New: JWT session token module to handle validation, information extraction, and exchange in to a Online Auth Token.
+- BREAKING: The JWT session token refactor changes how the exchange for online tokens happens and it can break use of the AuthShopSessionToken plug if you are relaying on it to redirect to the OAuth endpoint.
```

Comment:
```suggestion
- BREAKING: The JWT session token refactor changes how the exchange for online tokens happens and it can break use of the AuthShopSessionToken plug if you are relying on it to redirect to the OAuth endpoint.
```